src/confcom/azext_confcom/custom.py

```diff
@@ -40,27 +40,26 @@ def acipolicygen_confcom(
 ):
 
     if sum(map(bool, [input_path, arm_template, image_name])) != 1:
-        logger.error("Can only generate CCE policy from one source at a time")
-        sys.exit(1)
+        error_out("Can only generate CCE policy from one source at a time")
     if sum(map(bool, [print_policy_to_terminal, outraw, outraw_pretty_print])) > 1:
-        logger.error("Can only print in one format at a time")
-        sys.exit(1)
+        error_out("Can only print in one format at a time")
     elif (diff and input_path) or (diff and image_name):
-        logger.error("Can only diff CCE policy from ARM Template")
-        sys.exit(1)
+        error_out("Can only diff CCE policy from ARM Template")
     elif arm_template_parameters and not arm_template:
-        logger.error(
+        error_out(
             "Can only use ARM Template Parameters if ARM Template is also present"
         )
-        sys.exit(1)
+    elif save_to_file and arm_template and not (print_policy_to_terminal or outraw or outraw_pretty_print):
+        error_out("Must print policy to terminal when saving to file")
 
     if print_existing_policy:
-        if not arm_template:
-            logger.error("Can only print existing policy from ARM Template")
-            sys.exit(1)
-        else:
```

Comment:
Why removed this check?

Reply:
We were hitting the "can't have more than x statements in a function" style check so I moved that error checking inside of the `print_existing_policy_from_arm_template` function here: https://github.com/SethHollandsworth/azure-cli-extensions/blob/834fba5034189dea4df21e499997c872e24806f1/src/confcom/azext_confcom/template_util.py#L817 .
So it's still in there, just a different place